contrib/pipelines/secadvisor/README.md

```diff
@@ -5,30 +5,104 @@ Implements the IBM Security Advisor pipeline
 
 ## Setup Skydive
```

Comment:
Need to add before this a section that gives motivation to all of this text ... what we are trying to dfo here "Export flow information into Object Storage in such and such format, to be used by such and such .... " if we keep the name secadvisor in the path, maybe also reference to the relevant product pages directly.

Reply:
added this to the beginning of the file.